traffic_ops/v4-client/cachegroup_parameters.go

```diff
@@ -58,3 +58,10 @@ func (to *Session) CreateCacheGroupParameter(cacheGroupID, parameterID int, opts
 	reqInf, err := to.post(apiCachegroupParameters, opts, cacheGroupParameterReq, &data)
 	return data, reqInf, err
 }
+
+// CreateCacheGroupParameter associates a Parameter with a Cache Group.
```

Comment:
I believe this actually associates multiple Cache Groups with multiple Parameters

Reply:
updated the comments.